src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -52,7 +51,7 @@ class CDCL(val clauses: MutableList<MutableList<Int>>, initNumber: Int = 0) {
     )
 
     // convert values to a possible satisfying result: if a variable less than 0 it's FALSE, otherwise it's TRUE
-    private fun variableValues() = vars
+    protected fun variableValues() = vars
```

Comment:
Is this a `getModel` method?
Documentation is very unclear. How the void-arg method could "convert" anything? Maybe you wanted to say that this method "Returns the current assignment of (all) variables".

Where do you use this method? (i.e. "are you using it extensively?) I don't think this is `CDCL`'s responsibility to have this kinda method... Extension method (function) would be better.

Is it okay to return "true" (positive literal in your code) when variable status is not TRUE or FALSE? This doesn't look like an intuitive behavior, so this is not strictly a `getModel` method after all. The docs are unclear on this matter (the comment is technically correct, but only if you already know what is written(coded) inside).